internal/provider/models/log_streams_models.go

```diff
@@ -43,3 +44,10 @@ type SplunkHttpEventCollectorModel struct {
 	HecSourceName       types.String `tfsdk:"hec_source_name"`
 	Tls                 types.Bool   `tfsdk:"tls"`
 }
+
+type CrowdstrikeHttpEventCollectorModel struct {
+	HecHostPort   types.String `tfsdk:"hec_host_port"`
```

Comment:
```suggestion
	HecHostPort   types.String `tfsdk:"host_port"`
```
Please implement this fix on the Splunk model as well